lib/index.js

```diff
@@ -3055,6 +3059,254 @@ UFDS.prototype.listResolvers = function listResolvers(region, cb, noCache) {
     this.search(dn, opts, cb, noCache);
 };
 
+/**
+ * AccessKeys related methods
+ */
+
+/**
+ * Adds a new AccessKey to a given user record.
+ *
+ * The AccessKeyId and AccessKeySecret will be generated.
+ *
+ * This method will return you the full key as processed by UFDS.
+ *
+ * @param {Object} user the user record you got from getUser.
```

Comment:
It seems this also accepts a string (uuid?) for the user.

Same goes for following functions.

Reply:
If I remember correctly, this was intentionally not documented across the whole library in the past. Again, if I change this, I should probably go through the whole library and modify it for every method affected. Which I'm not opposed to do, though but, I'd also prefer to use different commits for these changes in order to make it clear why we're modifying each thing.

Reply:
Certainly.